docs/resources/schema-manager.md

```diff
@@ -0,0 +1,66 @@
+# DICOM Schema Manager
+
+#### What is it?
+Schema Manager is a command line app that upgrades the schema in your database from one version to the next. This requires migration scripts, starting with version 1, with each subsequent version providing both full and diff scripts. The typical migration script folder should look like this:
+- 1.sql
+- 2.sql
+- 2.diff.sql
+- 3.sql
+- 3.diff.sql
+- ...
+
+[DICOM Migration Scripts](https://github.com/microsoft/dicom-server/tree/main/src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations)
+
+------------
+
+#### How do you use it?
+DICOM Schema Manager currently has one command (**apply**) with the following options:
+
+| Option | Description |
+| ------------ | ------------ |
+| `-cs, --connection-string` | The connection string of the SQL server to apply the schema update. (REQUIRED) |
+| `-mici, --managed-identity-client-id` | The client ID of the managed identity to be used. |
+| `-at, --authentication-type` | The authentication type to use. Valid values are `ManagedIdentity` and `ConnectionString`. |
+| `-v, --version` | Applies versions between the current and specified version. |
```

Comment:
Current Schema Version. Since this was copy/pasted from the Resources file, I've updated both the Resources file and documentation to be more specific.